EESSI-pilot-install-software.sh

```diff
@@ -186,36 +186,42 @@ else
     echo_green ">> MODULEPATH set up: ${MODULEPATH}"
 fi
 
-for eb_version in '4.7.2' '4.8.0' '4.8.1'; do
+# assume there's only one diff file that corresponds to the PR patch file
+pr_diff=$(ls [0-9]*.diff | head -1)
+
+# use PR patch file to determine in which easystack files stuff was added
+for easystack_file in $(cat ${pr_diff} | grep '^+++' | cut -f2 -d' ' | sed 's@^[a-z]/@@g' | grep '^eessi.*yml$'); do
+
+    echo -e "Processing easystack file ${easystack_file}...\n\n"
+
+    # determine version of EasyBuild module to load based on EasyBuild version included in name of easystack file
+    eb_version=$(echo ${easystack_file} | sed 's/.*eb-\([0-9.]*\).*/\1/g')
```

Comment:
Alternative way might be

```bash
eb_version=$(echo ${easystack_file} | cut -f 4 -d '-')
```

Reply:
That's a bit less strict, at least with `eb-` we have some guarantee that which follows is an EasyBuild version...

So I prefer sticking to the `sed`.

Eventually we probably want to specify the EasyBuild version to use *in* the easystack file itself (but then EasyBuild needs to allow for that, I think).

Reply:
ack.